src/prommis/solid_handling/tests/test_crusher.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from pyomo.environ import ConcreteModel, assert_optimal_termination, value
+
+from idaes.core import FlowsheetBlock
+from idaes.core.initialization import (
+    BlockTriangularizationInitializer,
+    InitializationStatus,
+)
+from idaes.core.solvers import get_solver
+from idaes.core.util.model_diagnostics import DiagnosticsToolbox
+from idaes.core.util.model_statistics import (
+    number_total_constraints,
+    number_unused_variables,
+    number_variables,
+)
+from prommis.solid_handling.crusher import CrushAndBreakageUnit
+from prommis.solid_handling.crusher_solids_properties import CoalRefuseParameters
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver()
+
+
+# -----------------------------------------------------------------------------
+@pytest.mark.unit
+def test_config():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties_solid = CoalRefuseParameters(
+        doc="solid property",
+    )
+
+    m.fs.unit = CrushAndBreakageUnit(property_package=m.fs.properties_solid)
+
+    # Assert specific config options as per your model's requirements
+    # Example:
+    assert not m.fs.unit.config.dynamic
+    assert not m.fs.unit.config.has_holdup
+    assert m.fs.unit.config.property_package is m.fs.properties_solid
+
+
+# -----------------------------------------------------------------------------
+class TestSolidHandling(object):
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(dynamic=False)
+        m.fs.properties_solid = CoalRefuseParameters(
+            doc="solid property",
+        )
+
+        m.fs.unit = CrushAndBreakageUnit(property_package=m.fs.properties_solid)
+        # Set up your model initialization here
+        m.fs.unit.inlet.mass_frac_comp[0, :].fix(
+            0.1
+        )  # set mass frac value. There are 13 component in property package
+        m.fs.unit.properties_in[0].flow_mass.fix(2000)  # kg/hr
+        m.fs.unit.properties_in[0].particle_size_median.fix(80)  # micrometer
+        m.fs.unit.properties_in[0].particle_size_width.fix(1.5)  # dimensionless
+        m.fs.unit.properties_out[0].particle_size_median.fix(58)  # micrometer
+        m.fs.unit.properties_out[0].particle_size_width.fix(1.5)  # dimensionless
+        return m
+
+    @pytest.mark.build
+    @pytest.mark.unit
+    def test_build(self, model):
+        # Assertions related to the build status of the model
+        # More assertions as needed for your model
```

Comment:
You should remove these comments, and add some checks for the things you expect the model to build, i.e. `work`, `feed_p80`, `prod_p80`, and `crush_work_eq`.